build/optprof/UpdateRunSettings.ps1

```diff
@@ -14,11 +14,13 @@ $runsettingsXml = [Xml.XmlDocument](Get-Content $runsettingsPath)
 # https://stackoverflow.com/questions/33813700/empty-xml-node-rendered-as-string-in-powershell
 $testStores = $runsettingsXml.RunSettings.TestConfiguration.SelectSingleNode('TestStores')
 
-$profilingInputsStore = $runsettingsXml.CreateElement('TestStore')
+# https://stackoverflow.com/a/59090765/294804
+$profilingInputsStore = $runsettingsXml.CreateElement('TestStore', $runsettingsXml.DocumentElement.NamespaceURI)
 $profilingInputsStore.SetAttribute('Uri', "vstsdrop:$profilingInputsPath")
-$testStores.AppendChild($profilingInputsStore)
+$null = $testStores.AppendChild($profilingInputsStore)
```

Comment:
Is this just to emphasize that we're ignoring the result of `AppendChild`?

Reply:
Oh, this prevents the object returned from `AppendChild` from being output to the log?

Reply:
Correct (to the second question, not the first). It prevents putting the object returned into the output stream. One of the many things I learned about PowerShell from a previous team. 😝